packages/connect/src/types/api/bitcoin/index.ts

```diff
@@ -69,7 +67,6 @@ export interface SignTransaction {
     refTxs?: RefTransaction[];
     account?: {
         addresses: AccountAddresses;
-        utxo: AccountUtxo[];
```

Comment:
maybe also remove this line https://github.com/trezor/trezor-suite/blob/transport-refactor-2/packages/suite/src/actions/wallet/send/sendFormBitcoinActions.ts#L267